AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -32,6 +33,29 @@ @interface ASRangeController ()
 
 @implementation ASRangeController
 
+#pragma mark - NSObject
+
++ (void)load {
```

Comment:
Should use { on new line.  However, please just move this into the method below, where we create the weak set.  No need to observe the notification before the first controller is added to the set.